portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -2705,54 +3015,316 @@ definition {
-				readlessAction = "xx-ee",
-				type = "Link",
-				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
-		}
+	@description = "LPS-186875 Confirm that when clicking on the collapse buttons some fields are hidden"
+	@priority = 3
+	test TheCollapseButtonsHideFieldsOnCreationAction {
+		task ("And clicks on Create Item Action button") {
+			Button.click(button = "Creation Actions");
 
-		task ("When the user clicks on the new item action 3 dots button and edit") {
 			Click(
-				key_index = 2,
-				key_text = "Actions",
-				locator1 = "DataSet#VERTICAL_ELLIPSIS_N");
+				key_text = "New Creation Action",
+				locator1 = "CustomerPortalSite#BUTTON");
+		}
 
+		task ("When the user clicks on Display Options collapse button") {
 			Click(
-				key_menuItem = "Edit",
-				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+				key_componentName = "Display Options",
+				locator1 = "Panel#PANEL_TITLE");
 		}
 
-		task ("And go to Edit page for the action item created and leave the required fields, title and url empty") {
-			Type(
-				key_unnamedTitle = "",
-				locator1 = "TextInput#DEPOT_UNNAMED_DEFAULT_TITLE",
-				value1 = "");
+		task ("Then the Label and Icon fields are hidden") {
+			AssertNotVisible(
+				key_type = "Label",
+				locator1 = "DataSet#FILTER_SELECTION");
 
-			Type(
-				key_fieldId = "URL",
-				locator1 = "TextArea#ANY_ID",
-				value1 = "");
+			AssertNotVisible(
+				key_type = "Icon",
+				locator1 = "DataSet#FILTER_SELECTION");
 		}
 
-		task ("Then Error messages are displayed and the save button is disabled") {
-			AssertTextEquals(
-				locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
-				value1 = "This field is required.");
+		task ("Then the options Link, Modal and Side Panel are displayed in the modal") {
+			for (var typeOptions : list "Link,Modal,Side Panel") {
+				AssertElementPresent(
+					key_fieldLabel = "Type",
+					locator1 = "Select#GENERIC_SELECT_FIELD",
+					value1 = ${typeOptions});
+			}
+		}
 
-			AssertElementPresent(locator1 = "DataSet#ANY_BUTTON_DISABLED");
+		task ("Then clicks on Action Behavior and Type, URL and Headless Action Key fields are hidden") {
+			Click(
+				key_componentName = "Action Behavior",
+				locator1 = "Panel#PANEL_TITLE");
+
+			AssertNotVisible(
+				key_type = "Type",
+				locator1 = "DataSet#FILTER_SELECTION");
+
+			AssertNotVisible(
+				key_type = "URL",
+				locator1 = "DataSet#FILTER_SELECTION");
+
+			AssertNotVisible(
+				key_type = "Headless Action Key",
+				locator1 = "DataSet#FILTER_SELECTION");
 		}
 	}
 
-	@description = "LPS-194351 Confirm that when editing an already created modal action, it is not possible to leave the fields empty in the Creation Action"
-	@priority = 4
-	test TheModalActionFieldsCannotBeEmptyInCreationAction {
+	@description = "LPS-186875 Confirm that the Headles action key field is not mandatory"
+	@priority = 5
+	test TheHeadlesFieldIsNotMandatoryOnCreationAction {
```

Comment:
Name should be Headles**s**FieldIsNotMandatoryOnCreationAction (Dont to use "The" on every test name)